timesync/ticker_test.go

```diff
@@ -143,7 +144,12 @@ func TestTicker_timeSinceLastTick(t *testing.T) {
 		duration: 100 * time.Millisecond,
 		genesis:  c.Now().Add(-320 * time.Millisecond),
 	})
-	r.True(tr.timeSinceLastTick() >= 20)
+	// 20ms is the diff between time.now() and start of the current layer i.e. t.LayerToTime(currentLayer)
+	sinceStartOfLayer := 20 * time.Millisecond
+	r.True(tr.timeSinceLastTick() == sinceStartOfLayer)
```

Comment:
```suggestion
	r.Equal(tr.timeSinceLastTick(), sinceStartOfLayer)
```

Will produce nicer error messages.